### DISCUSSION TITLE
"message":"Cannot return null for non-nullable field PlaceBetResult.referenceId.",

### DISCUSSION THREAD

**Discussion by frafi1** [cloudbet/docs#50] [Q&A]
i have an internal server error since yesterday.
What is going wrong here? Has anything changed because this was okay last week?

My request: 
-------------------------------------------------------------------------------------------------------
const input = {
   referenceId: "ca398c64-2849-4eaf-9541-bc47695643f7",
   eventId: "16025258",
   marketUrl: "soccer.total_goals_period_second_half/under?total=1",
   currency: "PLAY_EUR",
   price: "2.02",
   stake: "1",
   side: "BACK",
   acceptPriceChange: "BETTER"
 };
 
The response:
-------------------------------------------------------------------------------------------------------
"errors":[
  {
   "message":"Cannot return null for non-nullable field PlaceBetResult.referenceId.",
   "locations":[
    {
     "line":3,
     "column":6
    }
   ],
   "path":[
    "placeBet",
    "referenceId"
   ],
   "extensions":{
    "code":"INTERNAL_SERVER_ERROR"
   }
  }
 ],
 "data":null
}

**Comment by kgravenreuth** [comment]
Hi @frafi1 
Thanks for highlighting this issue and apologies for the error seen. We're investigating this currently and we'll get back to you soon.


Cheers,
Klaus Gravenreuth
Product & Engineering @ Cloudbet

**Comment by kgravenreuth** [comment]
Hi @frafi1 

Apologies for the issue which was caused due to a misconfiguraton on our side. This should be fixed now. Please let us know if you still see any issues.

Cheers,
Klaus Gravenreuth
Product & Engineering @ Cloudbet

**Reply by jshelton31** [reply]
can you please advise on my new ticket, [https://github.com/Cloudbet/docs/discussions/66](https://github.com/Cloudbet/docs/discussions/66) thanks

**Comment by frafi1** [comment]
Hello Klaus,

it is okay now, everything works as expected, thanks.

Franz


  ----- Original Message -----
